spec/requests/users_controller_spec.rb

```diff
@@ -4612,7 +4621,7 @@ def create_totp
       it "creates a security key for the user" do
         simulate_localhost_webauthn_challenge
         create_second_factor_security_key
-        response_parsed = response.parsed_body
+        _response_parsed = response.parsed_body
```

Comment:
I think we can just remove this entire line?

Reply:
yeah I was just working through stuff mechanically to clear up warnings, did not want to amend functionality